src/components/styles/LinkSearch.css

```
@@ -0,0 +1,29 @@
+.LinkSearch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.LinkSearch > input {
+  box-sizing: border-box;
+  width: 1060px;
+  padding: 15px;
+  border-radius: 10px;
+  background: #f5f5f5;
+  border: none;
+  margin-left: 32px;
+  margin-right: 32px;
```

단축속성을 활용해서 더 간단하게 작성할 수 있습니다.
```suggestion
  margin: 0 32px;
```